script/publish_views

```diff
@@ -84,13 +94,24 @@ def main():
     except ValueError as e:
         parser.error(f"argument --log-level: {e}")
 
+    worker_entrypoint = partial(process_file, client, args)
+    sqlfiles = []
```

Comment:
Maybe `sql_files`?

Reply:
This is matching the naming used in `dryrun`. I will update the name in both places.